build-farm/platform-specific-configurations/linux.sh

```diff
@@ -41,7 +41,12 @@ fi
 
 if [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]
 then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched --enable-openssl-bundling"
+  if [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ]
+  then
+    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched"
```

Comment:
Deduplicate the `--with-openssl=fetched`

Reply:
Sorry, which is the duplicate you refer to?

Reply:
Code currently says
```if [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]
then
  if [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ]
  then
    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched"
  else
    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched --enable-openssl-bundling"
  fi
fi```
So in jdk12 we no longer need to bundle openssl (we will probably update jdk8 and jdk11 to the same options soon, but it requires some thought as to how to communicate the change to existing users).  It's meant to convey 'jdk12 differs from the other platforms in that it is built without --enable-openssl-bundling'.  Do you want me to rework it as
```
export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched"
  if [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ]
  then
    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --enable-openssl-bundling"
  fi
```
?

Reply:
Hi Simon, yes that reworking would be slightly cleaner.

Reply:
Rewritten as suggested.